core/templates/dev/head/pages/exploration-player-page/layout-directives/progress-nav.directive.ts

```diff
@@ -106,14 +106,14 @@ angular.module('oppia').directive('progressNav', [
           };
 
           var doesInteractionHaveSpecialCaseForMobile = function() {
-            // Special cases to show submit button:
-            // 1. ItemSelectionInput interaction with maximum selectable
-            //    choices > 1 in desktop/mobile mode.
-            // 2. ItemSelectionInput interaction with maximum selectable
-            //    choices == 1 in mobile mode.
-            // 3. MultipleChoiceInput interaction in mobile mode.
+            // The submit button should be shown:
+            // 1. In mobile mode, if the current interaction is either
+            //    ItemSelectionInput with maximum selectable choices == 1 or
+            //    MultipleChoiceInput.
```

Comment:
In mobile, the ItemSelectionInput can have one option and the submit button is still shown am I right? If yes please edit the comment.

Reply:
Yep, that is what "ItemSelectionInput with maximum selectable choices == 1" means (ie. in the customization arguments the creator has set maximum selectable as 1)

Reply:
Sorry, phrased it wrongly, when then number of choices for ItemSelectionInput is > 1 do we still show the submit button?

Reply:
When the number of choices is >1 we do show the submit button (in desktop and mobile). This is taken care of in the else part.

Reply:
But then don't mention in the comment that it is just for the cases when max selectable choices are 1, because it just makes it confusing.

Reply:
OK, but just to be clear. If it is mobile mode, and the choices are >1 it should fail the if condition and proceed to the else part.

Reply:
I still think we should keep it because it is possible that in future someone might try to simplify this if-else -- in that case this condition should be taken into account.